src/store/base-api-url.js

```diff
@@ -0,0 +1,3 @@
+export function getBaseApiUrl() {
+  return `http://localhost:5000`;
+}
```

Comment:
I would suggest to not place the url here, but to set the default base url for axios globally.
First, you can create env folder and create **config.local.json** file in it.
In the json file you can set the url.
Then, in App.vue on created() you can get this value and set the  axios.defaults.baseURL.